errors.cpp

```diff
@@ -152,6 +152,7 @@ static void ReplayErrorRecursive(const list_t *pList, bool fFatalOnly)
 		}
 		PrintSPASMError(lpError);
 	}
+	exit(-1);

```

Comment:
This should probably be a positive nonzero value, to avoid confusion with signal termination on unixes.